### DISCUSSION TITLE
Using a base stylesheet to be overwritten later with sync

### DISCUSSION THREAD

**Discussion by Trustadz** [six7/figma-tokens#1905] [Q&A] [answered]
So I might be thinking this all wrong but I'm trying to implement tokens into my figma workflow. As a little backstory: I create medium-high fidelity wireframes for multiple customers. Most of them start with a UI framework from which I will expand and detail to the clients branding.
I was hoping tokens would be able to help me with this but I'm getting stuck in my thinking.

Example:
I have a token (colors.primary) which an alias to {colors.indigo.600} and I have this library linked to my github to sync across all projects.
I would like to create a set per client where I can add new colors (so far so good) and then also update the colors.primary either by changing the value on only that project, or creating a new colors.primary in the new set and using that instead of the original colors.primary

I've going in circles with my thinking for a while now and it's not getting me anywhere, and no clue how to use the correct terms for finding an answer (something like replace, or overrule didn't gave me the results I thought I could use)

**Comment by six7** [comment, marked as answer]
Any reason why not use a set per client? https://docs.tokens.studio/themes/token-sets

**Reply by Trustadz** [reply]
As usual, the easiest answer is the correct one. Thanks! I didn't read the docs fully thinking I knew how it worked. This sentence fixed my issue.

"If you have multiple sets (e.g. a light and a dark theme) you can overwrite decisions that may have been defined in another set."
